README.md

```diff
@@ -29,6 +29,8 @@ server.post('/api/messages', (req, res) => {
 });
 ```
 
+The full quick start guide can be found in the [tyntec Docs Center](https://www.tyntec.com/docs/whatsapp-business-api-integration-microsoft-bot-framework).
```

Comment:
```suggestion
You can read a quick start guide in the [tyntec Docs Center](https://www.tyntec.com/docs/whatsapp-business-api-integration-microsoft-bot-framework).
```

Reply:
I would suggest using active voice here :)

Reply:
Like this: https://github.com/tyntec/botbuilder-adapter-tyntec-whatsapp/commit/4684ecea41632387405ccbce231be91141c5d4bf ?